src/docs/Dedication.adoc

```diff
@@ -2,13 +2,10 @@
 [discrete]
 == Legal notice
 
-+++ The Activeeon SAS, © 2007-<span id="currentYear">2019</span>. All Rights Reserved. +++
+The Activeeon SAS, © 2007-+++<span id="currentYear">2019</span>+++. All Rights Reserved.
```

Comment:
why `The Activeeon` ? why not just `Activeeon SAS`

Reply:
I think that Activeeon SAS is better. I'll apply the change.